auslib/db.py

```diff
@@ -946,7 +946,21 @@ def getReleaseInfo(self, product=None, limit=None,
             column = [self.name]
         else:
             column = [self.name, self.product, self.data_version, self.read_only]
+
         rows = self.select(where=where, columns=column, limit=limit, transaction=transaction)
+
+        if not nameOnly:
+            j = join(dbo.releases.t, dbo.rules.t, ((dbo.releases.name == dbo.rules.mapping) | (dbo.releases.name == dbo.rules.whitelist)))
+            ref_list = select([dbo.releases.name, dbo.rules.rule_id]).select_from(j).execute().fetchall()
+
+            for row in rows:
+                refs = [ref for ref in ref_list if ref[0] == row['name']]
+                ref_list = [ref for ref in ref_list if ref[0] != row['name']]
+                if len(refs) is not 0:
```

Comment:
You should probably use "> 0" here. "is" tests for identity, and I'm not sure Python guarantees that 0 always has the same id().